pkg/operator/tektonchains.go

```diff
@@ -29,9 +39,12 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
+// "quay.io/openshift-pipeline/chainstest"
+var repo string = os.Getenv("CHAINS_REPOSITORY")
+var tag string = time.Now().Format("010206150405")
+var public_key_path = resource.Path("testdata/chains/key")
```

Comment:
I guess that these should go to `const` similarly to https://github.com/openshift-pipelines/release-tests/blob/master/pkg/config/config.go#L17-L76